.github/workflows/bsim-tests-publish.yaml

```
@@ -25,4 +25,4 @@ jobs:
           commit: ${{ github.event.workflow_run.head_sha }}
           event_file: event/event.json
           event_name: ${{ github.event.workflow_run.event }}
-          files: "bsim-test-results/**/bsim_results.xml"
+          files: "bsim-*-test-results/**/bsim_results.xml"
```

Because we are now uploading both Bluetooth and Networking results as `bsim-test-results`.

```suggestion
          files: "bsim-test-results/**/bsim_results.xml"
```